src/Workspaces/CoreTest/SolutionTests/SolutionTests.cs

```diff
@@ -862,8 +862,10 @@ public async Task ChangingLanguageVersionReparses()
 
         [Theory]
         [InlineData("#if DEBUG", false, LanguageNames.CSharp)]
+        [InlineData(@"#region ""goo""", true, LanguageNames.CSharp)]
         [InlineData("// File", true, LanguageNames.CSharp)]
         [InlineData("#if DEBUG", false, LanguageNames.VisualBasic)]
+        [InlineData(@"#region ""goo""", true, LanguageNames.VisualBasic)]
```

Comment:
woot.  now documents with other pp directives can be reused when changing project configs.  In roslyn this is 4000+ docs.